products/appengine/api.yaml

```diff
@@ -67,30 +67,74 @@ objects:
           - ALLOW
           - DENY
   - !ruby/object:Api::Resource
-    name: 'Service'
+    name: 'ServiceSplitTraffic'
```

Comment:
Instead of renaming `Service`, would you mind duplicating this into a separate resource definition? It feels a little silly, but it's the best way to represent fine-grained resources right now.

Reply:
This will also require exclusions in Ansible / InSpec for the Magician to work.